### DISCUSSION TITLE
240330

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#95] [팀회의]
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항

## 어제 한 일
- 게시글 좋아요 기능
- 페이지 이름 통일
- 세 군데로 흩어져 있던 페이지 관련 상수 파일 하나로 통일 (src/constants/Page.ts)
- 페이지네이션 기본 구현

## 오늘 할 일
- 페이지네이션 바 기능 구현
- 페이지네이션 재사용 고려해서 데이터 기능 관련 파일 분리
- 공지글 페이지에 페이지네이션 적용
- 드롭다운 (편집/삭제) 컴포넌트 기능 추가
- 이슈 drag & drop 구현

## 오늘의 일정 (코어 타임)
- 🔥 열일 🔥

## 멘토님 질문 사항

**Comment by sobokki** [comment]
## 논의사항
## 어제 한 일
네브바 유저 프로필 있을경우 프로필 보여주고 없을 경우 기본 이미지 보여주기 ... (이것뿐 ㅜ)
## 오늘 할 일
-나의 페이지, 캘린더 페이지 , 주간 캘린더, 주간 팀캘린더 데이터 연동하기 ,
-월간 캘린더 스캐줄 2개까지만 보이고 더보기 기능으로 넣어주기 
-네브바 알림설정 기능 구현하기 
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항
teamContext, 프로필스택, 라우트 contants, 모달
## 어제 한 일
마이페이지 진행중
## 오늘 할 일
마이페이지
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
x
## 어제 한 일
memberList 수정
## 오늘 할 일
모달 찐 완성하기
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항
x
